mllib/src/main/scala/org/apache/spark/ml/Predictor.scala

```diff
@@ -59,10 +69,12 @@ private[ml] trait PredictorParams extends Params
 /**
  * :: DeveloperApi ::
  * Abstraction for prediction problems (regression and classification). It accepts all NumericType
- * labels and will automatically cast it to DoubleType in [[fit()]].
+ * labels and will automatically cast it to DoubleType in [[fit()]]. If this predictor support
```

Comment:
"If this predictor supports weights, it accepts all NumericType weights, which will be automatically cast to DoubleType in [[fit()]]." ?